lib/studies/crane/backdrop.dart

```diff
@@ -54,18 +93,27 @@ class _FrontLayer extends StatelessWidget {
               ),
             ),
           ),
-          child: ListView(
+          child: StaggeredGridView.countBuilder(
+            crossAxisCount: crossAxisCount,
+            crossAxisSpacing: 16.0,
             padding: isDesktop
                 ? EdgeInsets.symmetric(
-                    horizontal:
-                        isSmallDesktop ? appPaddingSmall : appPaddingLarge,
-                    vertical: 22)
-                : const EdgeInsets.all(20),
-            children: [
-              Text(title, style: Theme.of(context).textTheme.subtitle2),
-              const SizedBox(height: 20),
-              ItemCards(index: index),
-            ],
+                        horizontal:
+                            isSmallDesktop ? appPaddingSmall : appPaddingLarge)
+                    .add(EdgeInsets.only(bottom: widget.mobileTopOffset * 2))
+                : const EdgeInsets.symmetric(horizontal: 20)
+                    .add(EdgeInsets.only(bottom: widget.mobileTopOffset * 2)),
+            itemBuilder: (context, index) {
+              if (index == 0) {
+                return _header();
+              } else {
+                return DestinationCard(destination: destinations[index]);
+              }
+            },
+            staggeredTileBuilder: (index) => index == 0
+                ? StaggeredTile.fit(crossAxisCount)
+                : const StaggeredTile.fit(1),
+            itemCount: destinations.length,
```

Comment:
Shouldn't this be +1 for the header?

Reply:
Good find!

Reply:
It didn't work well, with it, not sure why, but keeping as is